src/mscorlib/shared/System/Memory.cs

```diff
@@ -115,27 +115,22 @@ public Memory(T[] array, int start, int length)
 
         /// <summary>
         /// Creates a new memory from a memory manager that provides specific method implementations beginning
-        /// at 'start' index and ending at 'end' index (exclusive).
+        /// at 0 index and ending at 'end' index (exclusive).
```

Comment:
Hmm... this comment didn't make sense to begin with (with the param names)

Reply:
See previous discussion on the phrasing in the comment:
https://github.com/dotnet/coreclr/pull/16658#discussion_r171326042